spec/models/subject_spec.rb

```diff
@@ -10,7 +10,8 @@
 require 'rails_helper'
 
 RSpec.describe Subject, type: :model do
-  subject { create(:subject) }
+  subject { create(:subject, subject_name: 'Mathematics') }
 
   it { should have_many(:courses).through(:course_subjects) }
+  its(:to_s) { should eq('Mathematics') }
```

Comment:
Just wondering humanize it?

Reply:
Que?

Reply:
C'est ce que je voulais dire ...

https://apidock.com/rails/String/humanize